infra/base-images/base-runner/run_fuzzer

```diff
@@ -23,20 +23,33 @@ cd $OUT
 
 FUZZER=$1
 shift
-CMD_LINE="$OUT/$FUZZER $FUZZER_ARGS $@"
 
-OPTIONS_FILE="${FUZZER}.options"
-if [ -f $OPTIONS_FILE ]; then
-  OPTIONS_ARGS=$(grep "=" $OPTIONS_FILE | sed 's/\(\w*\)\W*=\W*\(.*\)/-\1=\2 /g' | tr '\n' ' ')
-  CMD_LINE="$CMD_LINE $OPTIONS_ARGS"
-fi
+rm -rf /tmp/input/ && mkdir /tmp/input/
 
 SEED_CORPUS="${FUZZER}_seed_corpus.zip"
 if [ -f $SEED_CORPUS ]; then
   echo "Using seed corpus: $SEED_CORPUS"
-  rm -rf /tmp/seed_corpus/ && mkdir /tmp/seed_corpus/
-  unzip -d /tmp/seed_corpus/ $SEED_CORPUS > /dev/null
-  CMD_LINE="$CMD_LINE /tmp/seed_corpus/"
+  unzip -d /tmp/input/ $SEED_CORPUS > /dev/null
+  CORPUS=/tmp/input
+fi
+
+if [[ "$FUZZING_ENGINE" = afl ]]; then
+  export ASAN_OPTIONS="$ASAN_OPTIONS:abort_on_error=1:symbolize=0"
+  export MSAN_OPTIONS="$MSAN_OPTIONS:exit_code=86:symbolize=0"
```

Comment:
Why symbolize=0 ? What about adding symbolize=0 in LSAN_OPTIONS ?
Why not UBSAN_OPTIONS ?